doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_dynamodb_sink.md

```diff
@@ -29,41 +33,72 @@ Next sections will explain this in detail.
+
+[Top](#top)
+
+####<a name="section1.2.2"></a>DynamoDB tables naming conventions
+The name of these tables depends on the configured data model (see the [Configuration](#section2.1) section for more details):
+
+* Data model by service path (`data_model=dm-by-service-path`). As the data model name denotes, the notified FIWARE service path (or the configured one as default in [`NGSIRestHandler`](.ngsi_rest_handler.md)) is used as the name of the table. This allows the data about all the NGSI entities belonging to the same service path is stored in this unique table. The only constraint regarding this data model is the FIWARE service path cannot be the root one (`/`).
+* Data model by entity (`data_model=dm-by-entity`). For each entity, the notified/default FIWARE service path is concatenated to the notified entity ID and entityType in order to compose the table name. The concatenation character is `_` (underscore). If the FIWARE service path is the root one (`/`) then only the entity ID and type are concatenated.
+
+In both cases, the notified/defaulted FIWARE service is prefixed to the table name.
+
+It must be said DynamoDB [only accepts](http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/WorkingWithTables.html) alphanumerics and `_`, `-` and `.`. All the other characters will be escaped to underscore (`_`) when composing the table names.
+
+The following table summarizes the table name composition:
+
+| FIWARE service path | `dm-by-service-path` | `dm-by-entity` |
+|---|---|---|
+| `/` | `<svc>` | `<svc>_<entityId>_<entityType>` |
+| `/<svcPath>` | `<svc>_<svcPath>` | `<svc>_<svcPath>_<entityId>_<entityType>` |
+
+Please observe the concatenation of entity ID and type is already given in the `notified_entities`/`grouped_entities` header values (depending on using or not the grouping rules, see the [Configuration](#section2.1) section for more details) within the Flume event.
+
+[Top](#top)
+
+####<a name="section1.2.3"></a>Row-like storing
+Regarding the specific data stored within the datastore associated to the resource, if `attr_persistence` parameter is set to `row` (default storing mode) then the notified data is stored attribute by attribute, composing an insert for each one of them. Each insert contains the following fields:
 
-* `table-by-destination`. A table named as the concatenation of `<fiware-service>_<fiware_servicePath>_<destination>` is created (if not yet existing). These values are notified as http headers.
-* `table-by-service-path`. A table named as the `<fiware-service>_<fiware-servicePath>` is created (if not yet existing). These values are notified as http headers.
+* `recvTimeTs`: UTC timestamp expressed in miliseconds.
+* `recvTime`: UTC timestamp in human-redable format ([ISO 8601](http://en.wikipedia.org/wiki/ISO_8601)).
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1042/commits/3913c56cb58a1dea3f077c77817cdce9080c530f